### PR TITLE
Issue: Fix broken links on blog entries and doc page

### PR DESCRIPTION
**Pull request by pjcalvo**
## Proposed changes

I found a couple of broken links on some old posts and a wrong reference to a non-existent selenium page on the uploadFile docs.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-sync/webdriverio-core.d.ts [hunk 1]
```diff
@@ -1207,7 +1207,7 @@ declare namespace WebdriverIO {
 
         /**
          * Uploads a file to the Selenium Standalone server or other browser driver
-         * (e.g. Chromedriver) by using the [`file`](docs/api/selenium.html#file) command.
+         * (e.g. Chromedriver).
```
Comment (christian-bromann) [inline packages/wdio-sync/webdriverio-core.d.ts:1210]: The new location is here: https://webdriver.io/docs/api/selenium.html#file

File: docs/Frameworks.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ it('should test something', () => {
 })
 ```
 
-If you want to run something asynchronously, you can either use the [`browser.call`](call) command or [custom commands](CustomCommands.md).
+If you want to run something asynchronously, you can either use the [`browser.call`](api/browser/call.html) command or [custom commands](CustomCommands.md).
```
Comment (christian-bromann) [inline docs/Frameworks.md:54]: This will be transformed by Docusarus, no need for this change

File: docs/SyncVsAsync.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ await el.click()
 
 ## Sync mode
 
-If you're using [`@wdio/sync`](https://www.npmjs.com/package/@wdio/sync) then you can avoid awaiting for browser/element calls. It is still required to deal with Promises from 3rd-party libraries, you should use [browser.call](docs/api/browser/call.html) for this.
+If you're using [`@wdio/sync`](https://www.npmjs.com/package/@wdio/sync) then you can avoid awaiting for browser/element calls. It is still required to deal with Promises from 3rd-party libraries, you should use [browser.call](api/browser/call.html) for this.
```
Comment (christian-bromann) [inline docs/SyncVsAsync.md:47]:
```suggestion
If you're using [`@wdio/sync`](https://www.npmjs.com/package/@wdio/sync) then you can avoid awaiting for browser/element calls. It is still required to deal with Promises from 3rd-party libraries, you should use [browser.call](call) for this.
```

File: docs/Frameworks.md (hunk 2 above)
Comment (pjcalvo) [reply docs/Frameworks.md:54]: I didn't change them.. This is exactly how it is on the repo. I am looking into why this looks like it changed. :S

Comment (christian-bromann) [reply docs/Frameworks.md:54]: nevermind then

File: packages/wdio-sync/webdriverio-core.d.ts (hunk 1 above)
Comment (christian-bromann) [reply packages/wdio-sync/webdriverio-core.d.ts:1210]: should we add this link then?

**Comment by christian-bromann** [comment]
ping @pjcalvo

**Comment by pjcalvo** [comment]
> ping @pjcalvo

@christian-bromann Sorry I have been dragged into many things lately. I will get this fixed before next week.

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**